T4298: solve ansible warnings

hakwerk opened the pull request:
Fix group name and remove obsolete empty command to remove the warnings from ansible.

dmbaturin commented:
Hi @hakwerk,
Thanks for your contribution! Could you please create a task in https://phabricator.vyos.net and reference it in the commit id, like "Txxxx: fix group name and remove..."?

Please look at PRs in vyos/vyos-1x for good examples.

hakwerk committed:
T4298: fix ansible group name and remove obsolete empty command

After commit 680a2b4 ("fix errors in custom package install") an empty
"command:" remained.

hakwerk committed:
T4298: fix ansible group name and remove obsolete empty command

Ansible does not like the dash in the group name
